src/nyc_trees/apps/event/tests.py

```diff
@@ -135,3 +138,20 @@ def test_rsvp_limit_increase(self):
         self.event = Event.objects.get(id=self.event.id)
         self.assertEqual(15, context['max_attendees'])
         self.assertEqual(15, self.event.max_attendees)
+
+    def test_starting_soon(self):
```

Comment:
This is an excellent set of test cases. The only drawback is that they don't verify that the `starting_soon == True` for the entire length of the event, only the exact moment of start. Consider adding one more assertion for `hour=12, minute=n` where `0 < n < 60`